bases/br_me_comex_stat/municipio_exportacao/table_config.yaml

```diff
@@ -100,48 +101,48 @@ columns: # REQUIRED
   
     -   
         name: id_sh4
-        description: <descrição da coluna>
+        description: ID Sistema Harmonizado
         is_in_staging: True # Bool [True, False], whether the column is in the staging table
         is_partition: False # Bool [True, False], whether the column is a partition.
   
     -   
         name: id_pais
-        description: <descrição da coluna>
+        description: ID País
         is_in_staging: True # Bool [True, False], whether the column is in the staging table
         is_partition: False # Bool [True, False], whether the column is a partition.
   
     -   
         name: sigla_uf
-        description: <descrição da coluna>
+        description: Sigla da Unidade da Federação
```

Comment:
Ricardo, em algumas tabelas está escrito "Sigla do Estado", temos que homogeneizar, não sei pra qual das duas rs

Reply:
Onde? Não nessas tabelas aqui né? Se for nas outras do repositório, com certeza. Estamos com tarefas de revisão no Trello que vamos pegar daqui a pouco.